### ISSUE TITLE
Need to Run "make" Twice

### ISSUE THREAD

**Issue by MikaelHashiCorp** [fhemberger/nomad-demo#8]
## PROBLEM:

When I run `make` once, the end result is broken Nomad server build.  Nomad is only installed on `consul-nomad-node1`, leaving it leaderless:  

```
vagrant@consul-nomad-node1:~$ nomad server members
Name                       Address        Port  Status  Leader  Protocol  Build  Datacenter  Region
consul-nomad-node1.global  192.168.56.21  4648  alive   false   2         1.2.6  dc1         global

Error determining leaders: 1 error occurred:
	* Region "global": Unexpected response code: 500 (No cluster leader)

vagrant@consul-nomad-node1:~$ find / -iname "nomad" -print 2>/dev/null
/usr/bin/nomad
/sys/fs/cgroup/cpuset/nomad
/vagrant/roles/nomad
/var/lib/nomad

vagrant@consul-nomad-node1:~$ nomad version
Nomad v1.2.6 (a6c6b475db5073e33885377b4a5c733e1161020c)
```

Nomad is not installed on `consul-nomad-node2 ` or `consul-nomad-node3 `:
```
vagrant@consul-nomad-node2:~$ nomad version

Command 'nomad' not found, but can be installed with:

apt install nomad
Please ask your administrator.

vagrant@consul-nomad-node2:~$ ps -ef | head -1 && ps -ef | grep nomad | grep -v grep
UID          PID    PPID  C STIME TTY          TIME CMD
vagrant@consul-nomad-node2:~$ find / -iname "nomad" -print 2>/dev/null
/vagrant/roles/nomad

vagrant@consul-nomad-node3:~$ find / -iname "nomad" -print 2>/dev/null
/vagrant/roles/nomad
vagrant@consul-nomad-node3:~$ 
```

## WORKAROUND SOLUTION:

Run `make` again.  This will finish up missed provisioning of the servers.  After the second `make` run, Nomad is healthy:
```
vagrant@consul-nomad-node1:~$ nomad server members
Name                       Address        Port  Status  Leader  Protocol  Build  Datacenter  Region
consul-nomad-node1.global  192.168.56.21  4648  alive   false   2         1.2.6  dc1         global
consul-nomad-node2.global  192.168.56.22  4648  alive   true    2         1.2.6  dc1         global
consul-nomad-node3.global  192.168.56.23  4648  alive   false   2         1.2.6  dc1         global
vagrant@consul-nomad-node1:~$ nomad status
No running jobs
vagrant@consul-nomad-node1:~$ 
```

**Comment by fhemberger** [comment]
Thanks for reporting, will look into it …

**Comment by fhemberger** [comment]
Ansible didn't wait for Vault nodes to be unsealed, fixed in 65909cc.
